manifests/install.pp

```diff
@@ -80,12 +80,10 @@
 
       # Puppet Enterprise 3.8 and ships an embedded r10k so thats all thats supported
       # This conditional should not effect FOSS customers based on the fact
-      unless ($::is_pe == 'true' or $::is_pe == true) and versioncmp("${::pe_version}", '3.8.0') >= 0 and $puppet_master { #lint:ignore:only_variable_string
```

Comment:
This was one of those "why did it ever pass?" tests. Elsewhere, references were made to `Package[$real_package_name]`, though because of the conditional it might not be present in the catalog, which caused failures. Even though PE 3.8 ships with r10k and the package is already installed, it still needs to be present in the resulting catalog - and this will restore the gem if someone mucks with the puppetserver gem environment.